server/utils/bookingUtils.ts

```diff
@@ -117,6 +117,13 @@ export const deriveBookingHistory = (booking: Booking) => {
   return history
 }
 
+export const shortenedOrExtended = (extension: Extension): 'shortened' | 'extended' => {
+  const previousDepartureDate = DateFormats.convertIsoToDateObj(extension.previousDepartureDate)
+  const newDepartureDate = DateFormats.convertIsoToDateObj(extension.newDepartureDate)
+
+  return previousDepartureDate.getTime() > newDepartureDate.getTime() ? 'shortened' : 'extended'
```

Comment:
Shame we have to convert to date and then into time for the comparison. I assume we can't go straight into a time object.

Reply:
I had a quick look into this - _in theory_, if we can guarantee there will never be a timezone marker, we can sort the date strings lexically. But that feels unsafe and I'd rather err on the side of caution here